python/langsmith/wrappers/_openai.py

```diff
@@ -382,5 +391,41 @@ def wrap_openai(
             tracing_extra=tracing_extra,
             invocation_params_fn=functools.partial(_infer_invocation_params, "chat"),
         )
+    # For the responses API: "client.responses.create(**kwargs)"
+    if hasattr(client, "responses"):
+
+        def process_outputs(response: Any):
+            if response:
+                try:
+                    return {
+                        "output_text": response.output_text,
+                        **response.model_dump(mode="json"),
+                    }
+                except Exception:
+                    return {"output": response}
+            return {}
+
+        if hasattr(client.responses, "create"):
+            client.responses.create = _get_wrapper(  # type: ignore[method-assign]
+                client.responses.create,
+                "openai.responses.create",
+                _reduce_response,
+                process_outputs=process_outputs,
+                tracing_extra=tracing_extra,
+                invocation_params_fn=functools.partial(
+                    _infer_invocation_params, "chat"
+                ),
+            )
+        if hasattr(client.responses, "parse"):
+            client.responses.parse = _get_parse_wrapper(  # type: ignore[method-assign]
+                client.responses.parse,
+                "openai.responses.parse",
+                tracing_extra=tracing_extra,
+                invocation_params_fn=functools.partial(
+                    _infer_invocation_params, "chat"
+                ),
+            )
+
+        # TODO: Handle stream() correctly. It's a context manager.
```

Comment:
hopefully can copy a lot of wrap_anthropic

Reply:
Ya both would use the same primitive i think